README.md

```diff
@@ -12,6 +12,8 @@
 </p>
 
 > Buefy is a lightweight library of responsive UI components for [Vue.js](https://vuejs.org/) based on [Bulma](http://bulma.io/) framework and design.
+> 
+> *Check out the official [`@ntohq/Buefy-next`](https://github.com/ntohq/buefy-next) fork for `Vue v3.0.0+` support.*
```

Comment:
Do we have to mention that the buefy-next project is not finished?

Reply:
I could add "fork in development for Vue 3.0.0+ support